hawc/apps/assessment/views.py

```diff
@@ -419,6 +420,20 @@ class AssessmentValueDetail(BaseDetail):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["adaf_footnote"] = constants.ADAF_FOOTNOTE
+        # add a message if POD/UF is not equal to the value
+        obj = self.object
+        if obj.value and obj.pod_value and obj.uncertainty:
+            calc_value = obj.pod_value / obj.uncertainty
+            if not np.isclose(
+                obj.value,
+                calc_value,
+                rtol=0.01,
+            ):
+                context["pod_message"] = f"""The POD/UF = Value is different than what is reported.
```

Comment:
the 0.364 is hard coded; it should be `calc_value`. Make `pod_message` a dictionary and then we can edit the text in the template instead of the view. 

We may want to also add to the message something like "Note: this message is only shown to team members and higher" (and make sure that is the case).